doc/rbd/rbd-encryption.rst

```diff
@@ -252,6 +252,13 @@ Here is an example of a command that maps a formatted clone:
 
    rbd device map -t nbd -o encryption-passphrase-file=clone-passphrase.bin -o encryption-passphrase-file=passphrase.bin mypool/myclone
 
+If the parent image and the clone are formatted with different encryption
+formats, use a command of this form:
+
+.. prompt:: bash #
+
+   rbd device map -t nbd -o encryption-format=luks2,encryption-passphrase-file=clone-passphrase.bin,encryption-format=luks1,encryption-passphrase-file=passphrase.bin mypool/myclone
```

Comment:
I don't think this is needed. `encryption-format` option default works for both LUKS1 and LUKS2, so the shorter variant that specifies only `encryption-passphrase-file` options would work just fine.

Reply:
Let's get rid of this paragraph and just merge `encryption-passphrase-file` options in the above example under a single `-o` -- a one-line change ;)

Reply:
@idryomov: I think I've done it the way you've asked, but let me know if I need to change it.